[2023-09-04T06:02:52.216+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2023-09-04T06:02:52.229+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2023-09-04T06:02:52.230+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:52.249+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-28 00:00:00+00:00
[2023-09-04T06:02:52.257+0000] {standard_task_runner.py:57} INFO - Started process 5545 to run task
[2023-09-04T06:02:52.260+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-28T00:00:00+00:00', '--job-id', '530', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmppqp_eyr6']
[2023-09-04T06:02:52.262+0000] {standard_task_runner.py:85} INFO - Job 530: Subtask insert_into_table
[2023-09-04T06:02:52.320+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-28T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:52.431+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-28T00:00:00+00:00'
[2023-09-04T06:02:52.433+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-28', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:02:52.443+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:52.453+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:52.465+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-28', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:02:52.468+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:02:52.482+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220428T000000, start_date=20230904T060252, end_date=20230904T060252
[2023-09-04T06:02:52.512+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:52.529+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:42.191+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2023-09-04T06:05:42.226+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2023-09-04T06:05:42.227+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:42.450+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-28 00:00:00+00:00
[2023-09-04T06:05:42.483+0000] {standard_task_runner.py:57} INFO - Started process 6882 to run task
[2023-09-04T06:05:42.502+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-28T00:00:00+00:00', '--job-id', '1088', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp8a5gxoh8']
[2023-09-04T06:05:42.512+0000] {standard_task_runner.py:85} INFO - Job 1088: Subtask insert_into_table
[2023-09-04T06:05:42.727+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-28T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:43.014+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-28T00:00:00+00:00'
[2023-09-04T06:05:43.017+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-28', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:43.039+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:43.063+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:43.091+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-28', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:43.101+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:43.148+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220428T000000, start_date=20230904T060542, end_date=20230904T060543
[2023-09-04T06:05:43.257+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:43.574+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
